src/field.h

```diff
@@ -64,12 +64,12 @@ static int secp256k1_fe_is_zero(const secp256k1_fe *a);
 /** Check the "oddness" of a field element. Requires the input to be normalized. */
 static int secp256k1_fe_is_odd(const secp256k1_fe *a);
 
-/** Compare two field elements. Requires magnitude-1 inputs. */
+/** Compare two field elements. Requires the first input to have magnitude equal to 1. 
+ *  The variable time counterpart of this function `secp256k1_fe_equal_var` was removed 
+ *  since it hits fast path only when the inputs are unequal. This unequal inputs 
```

Comment:
I think this is something you can document in the commit comment; it's weird to have it in the codebase. As in: there are many other things that aren't implemented because they're not worth it; they aren't exhaustively listed in the source code.